### PR TITLE
Il/1387 add voting education page

### PR DESCRIPTION
**Pull request by ilodidoj**
[Link to  issue.](https://github.com/usdoj-crt/crt-portal-management/issues/1387)

## What does this change?
New voting resource page created. This can be accessed from [siteroot]/voting_resources.
This will be integrated as a link in the voting toggle PR later.

### Author

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for, document, and establish a testing plan for any behavior that may vary across environments or is otherwise difficult to test.
+ [x] Check for [accessibility](https://github.com/usdoj-crt/crt-portal/blob/develop/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [ ] Check for any behavior that may vary across environments or is difficult to test, and ensure that it is well-understood, documented, and that there is a testing plan in place.
+ [ ] Re-check for [accessibility](https://github.com/usdoj-crt/crt-portal/blob/develop/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

This is a standalone page available at [siteroot]/voting-resources

### CODE REVIEW

File: crt_portal/crt_portal/urls.py [hunk 1]
```diff
@@ -81,6 +81,7 @@
     path('privacy-policy', TemplateView.as_view(template_name="privacy.html"), name='privacy_policy'),
     path('hate-crime-human-trafficking', TemplateView.as_view(template_name="hate_crime_human_trafficking.html"), name='hate_crime_human_trafficking'),
     path('housing-resources', TemplateView.as_view(template_name="hce_resources.html"), name='hce_resources'),
+    path('voting-resources', TemplateView.as_view(template_name="vot_resources.html"), name='vot_resources'),
```
Comment (billyhunt) [inline crt_portal/crt_portal/urls.py:84]: We might consider renaming the template voting_resources.html for consistency.

File: crt_portal/cts_forms/templates/vot_resources.html [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+            <li>{% trans "Denial or lack of accessibility while voting;" %}</li>
+            <li>{% trans "Military or overseas absentee voting issues." %}</li>
+          </ul>
+        </section>
+        <section id="voting-rights">
+          <h2>{% trans "Learn about my voting rights" %}</h2>
+
+          <p>
+            <ul>
+            <li>{% trans "Want to know your federal rights? Download the Justice Department's new " %}<a class="link--blue" href="https://www.justice.gov/file/1517936/download">Know Your Voting Rights</a> guide.
```
Comment (billyhunt) [inline crt_portal/cts_forms/templates/vot_resources.html:93]: Should "Know Your Voting Rights" be translated?

File: crt_portal/cts_forms/templates/vot_resources.html [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+            </li>
+            <li>{% trans "Do you have a criminal conviction? Wondering if you can vote in your state?<br>Download the Justice Department's new " %}<a class="link--blue" href="https://www.justice.gov/voting/file/1507306/download">Guide to State Voting Rights that Apply After Criminal Conviction</a></li>
```
Comment (billyhunt) [inline crt_portal/cts_forms/templates/vot_resources.html:95]: Should "Guide to State Voting Rights that Apply After Criminal Conviction" be translated?

File: crt_portal/cts_forms/templates/vot_resources.html [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+            <li>{% trans "Learn about federal laws that protect voters with disabilities at " %}<a class="link--blue" href="https://ada.gov">ada.gov.</a></li>
```
Comment (billyhunt) [inline crt_portal/cts_forms/templates/vot_resources.html:96]: "." should not be in link in ada.gov

change to 

```
<li>{% trans "Learn about federal laws that protect voters with disabilities at " %}<a class="link--blue" href="https://ada.gov">ada.gov</a>.</li>
```

File: crt_portal/cts_forms/templates/vot_resources.html [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+            <li>{% trans "Find additional information about voting rights at " %}<a class="link--blue" href="https://justice.gov">justice.gov.</a></li>
```
Comment (billyhunt) [inline crt_portal/cts_forms/templates/vot_resources.html:97]: Same as above.  Consider moving '.' outside of <a> tag.

File: crt_portal/cts_forms/templates/vot_resources.html [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+            <li>{% trans "Threat against voters" %}</li>
+            <li>{% trans "Threat against election officials" %}</li>
+            <li>{% trans "Election fraud" %}</li>
+          </ul>
+        </section>
+        <section id="polling-place">
+          <h2>{% trans "Find polling place location and hours" %}</h2>
+
+          <p>
+            {% trans "Go to" %} <a class="external-link--blue" href="https://usa.gov/election-office">usa.gov/election-office</a>  {% trans " to find your state election website." %}
```
Comment (billyhunt) [inline crt_portal/cts_forms/templates/vot_resources.html:68]: Does this need an external link modal?

File: crt_portal/cts_forms/templates/vot_resources.html [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+        <h1 id="get-help">
+          {% trans "Report voting issues" %}
+        </h1>
+        <div class="crt-landing--separator"></div>
+
+        <section id="criminal-violation">
+          <h2>{% trans "Report a possible federal criminal violation" %}</h2>
+          
+          <p>
+            {% trans "To report a possible federal crime, go to" %} <a class="external-link--blue" href="https://tips.fbi.gov">tips.fbi.gov</a>  {% trans " or call 800-CALL-FBI." %}
```
Comment (billyhunt) [inline crt_portal/cts_forms/templates/vot_resources.html:53]: Does this need an external link modal?

File: crt_portal/cts_forms/templates/vot_resources.html [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+{% endblock %}
+
+{% block content %}
+<div class="crt-landing--section crt-landing--pale crt-subpage">
+  <div class="grid-container">
+    <div class="grid-row grid-gap">
+      <div class="tablet:grid-col-4">
+        <ul class="usa-sidenav crt-subpage--nav" id="toc">
+          <li class="usa-sidenav__item">
+            <a href="#get-help">{% trans "How Do I..." %}</a>
```
Comment (billyhunt) [inline crt_portal/cts_forms/templates/vot_resources.html:25]: Should this be changed to "Report Voting Issues" to mirror the housing resources page?

**Review by billyhunt — CHANGES_REQUESTED** [review]
Looks really good!  Just a couple of small things to consider.

File: crt_portal/cts_forms/templates/vot_resources.html (hunk 2 above)
Comment (ilodidoj) [reply crt_portal/cts_forms/templates/vot_resources.html:93]: Yes translation will be later part when we have the final version of the page.

File: crt_portal/cts_forms/templates/vot_resources.html (hunk 4 above)
Comment (ilodidoj) [reply crt_portal/cts_forms/templates/vot_resources.html:96]: All the punctuations will be changed accordingly when we get final version of this page in the new PR.

File: crt_portal/crt_portal/urls.py (hunk 1 above)
Comment (ilodidoj) [reply crt_portal/crt_portal/urls.py:84]: We are following the pattern from the hec_resources.  This is consistent with the previous pattern.

File: crt_portal/cts_forms/templates/vot_resources.html (hunk 6 above)
Comment (ilodidoj) [reply crt_portal/cts_forms/templates/vot_resources.html:68]: this is external to DOJ, but still federal site, so no modal.

File: crt_portal/cts_forms/templates/vot_resources.html (hunk 7 above)
Comment (ilodidoj) [reply crt_portal/cts_forms/templates/vot_resources.html:53]: within same federal space, no modal.

**Review by billyhunt — APPROVED** [review]
Looks good to me.

**Pull request merged**